universal-login-react/src/ui/TopUp/Fiat/FiatFooter.tsx

```diff
@@ -29,6 +30,9 @@ export const FiatFooter = ({paymentMethod}: FiatFooterProps) => {
           <div className="info-block info-row">
             <p className="info-text info-text-hint">Minimum amount is 1£</p>
           </div>
+          <div className="info-block info-row">
+            <p className="info-text info-text-hint">Minimum amount required to deploy contract is {minimalAmount} ETH</p>
+          </div>
```

Comment:
We should do not show this info if it is not deployment and probably use same info as in crypto (DeploymentWithCryptoInfo)

Reply:
👍